src/praisonai/tests/unit/test_sequential_tool_calling.py

```
@@ -0,0 +1,679 @@
+        mock_completion.side_effect = responses
+        
+        # Create agent with streaming
+        agent = Agent(
+            instructions="You are a helpful assistant.",
+            llm='gpt-3.5-turbo',
+            tools=[get_stock_price, multiply],
+            stream=True
+        )
+        
+        # Execute
+        result = agent.start("Get Tesla stock price and multiply by 2.")
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)